### PR TITLE
limit book title to 25 chars

### PR DESCRIPTION
**Pull request by sebasdeldi**
## Overview

- Truncates existing book title strings to 25 chars

- Limits book title strings to 25 chars

## Card
https://trello.com/c/bGJmUINd/44-migraciones-opcional

### CODE REVIEW

File: db/migrate/20190318130015_change_book_title_length.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class ChangeBookTitleLength < ActiveRecord::Migration[5.1]
+  def change
+    Book.find_each do |book|
```
Comment (nicozar95) [inline db/migrate/20190318130015_change_book_title_length.rb]: Maybe we can move this logic to a rake task or something similar.

File: db/migrate/20190318130015_change_book_title_length.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      if book.title.length > 25
+        book.update(title: book.title.truncate(25))
```
Comment (nicozar95) [inline db/migrate/20190318130015_change_book_title_length.rb]: Beware about this logic, you dont have to truncate the title to 25 characters, If the title surpass 25 chars it should be 22 chars + '...'

Comment (nicozar95) [reply db/migrate/20190318130015_change_book_title_length.rb]: Example: 
```
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam
```
This paragraph has 202 characters which is far from the limit of 25, it should be replaced with:
```
Lorem ipsum dolor sit ...
```

File: db/migrate/20190318130015_change_book_title_length.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+      end
+    end
+    change_column :books, :title, :string, :limit => 25
```
Comment (nicozar95) [inline db/migrate/20190318130015_change_book_title_length.rb]: Can you use the new ruby notation for this? `limit: 25`

**Review by nicozar95 — CHANGES_REQUESTED** [review]
We could add some test for this

File: db/migrate/20190318130015_change_book_title_length.rb (hunk 2 above)
Comment (sebasdeldi) [reply db/migrate/20190318130015_change_book_title_length.rb]: .truncate(n) does exactly that.

**Review by nicozar95 — APPROVED** [review]

**Pull request merged**